sadcowpng/sharedspaces/tests.py

```diff
@@ -1,3 +1,389 @@
+
+        # put the data into the space mode and create a new space model
+        test_space = Space(space_name=name,
+                           space_description=description,
+                           space_max_capacity=max_capacity,
+                           space_noise_level_allowed=noise_level_allowed,
+                           space_noise_level=noise_level,
+                           space_wifi=wifi,
+                           space_restrooms=restroom,
+                           space_food_drink=food_drink)
+
+        # Submitting test form data to the create space database
+        test_space.save()
+
+        # Running tests to see accurate data made it's way to the database
+        # while doing so check the string output as well
+        # Tests to see if all the information from the string return is accurate
+
+        # Test name
+        self.assertEqual(test_space.name_str(), name, 'The location name was stored in the database incorrectly.')
+
+        # Deleting this entry from the database with it's unique ID number
+        test_space.delete()
+
+        # now check if it was deleted properly
+        self.assertEqual(test_space.pk, None,
+                         'The location was not deleted properly from the database.')
+
+    # Tests that cover database accuracy once form is submitted
+    def test_form_to_database_accuracy_description(self):
+        # first save the data in to the database
+        # start by extracting the data from the form
+        name = TestCase.test_form.cleaned_data['space_name']
+        description = TestCase.test_form.cleaned_data['space_description']
+        max_capacity = TestCase.test_form.cleaned_data['space_max_capacity']
+        noise_level_allowed = int(TestCase.test_form.cleaned_data["space_noise_level_allowed"][0])
+        noise_level = int(TestCase.test_form.cleaned_data["space_noise_level"][0])
+        wifi = TestCase.test_form.cleaned_data['space_wifi']
+        restroom = TestCase.test_form.cleaned_data['space_restrooms']
+        food_drink = TestCase.test_form.cleaned_data['space_food_drink']
+
+        # put the data into the space mode and create a new space model
+        test_space = Space(space_name=name,
+                           space_description=description,
+                           space_max_capacity=max_capacity,
+                           space_noise_level_allowed=noise_level_allowed,
+                           space_noise_level=noise_level,
+                           space_wifi=wifi,
+                           space_restrooms=restroom,
+                           space_food_drink=food_drink)
+
+        # Submitting test form data to the create space database
+        test_space.save()
+
+        # Running tests to see accurate data made it's way to the database
+        # while doing so check the string output as well
+        # Tests to see if all the information from the string return is accurate
```

Comment:
I've noticed that you've repeated this same block of comments. If that's the case, maybe you should take it out of both locations and instead write about it in either the README or above the class by describing the testing flow.

Reply:
Removed to a central location at the beginning of all database accuracy test.